kge/model/embedder/lookup_embedder.py

```diff
@@ -58,7 +57,7 @@ def __init__(
                 dropout = 0
         self.dropout = torch.nn.Dropout(dropout)
 
-    def prepare_job(self, job: Job, **kwargs):
```

Comment:
I had to remove the Job dependency, due to circular imports (I had to add KgeModel to Job). Do you have a better idea than just to remove it?
Dependency Injection seems a little overkill for the little type annotation.

Reply:
Perhaps you can import KgeModel in Job only where needed?

Reply:
done